numpy/core/fromnumeric.py

```diff
@@ -1826,15 +1835,14 @@ def sum(a, axis=None, dtype=None, out=None, keepdims=False):
             sum = a.sum
         except AttributeError:
             return _methods._sum(a, axis=axis, dtype=dtype,
```

Comment:
Sorry to raise a stylistic point at this late stage, but since the `return _methods._sum` call is now identical to the one at the end, I would suggest rewriting this (as well as other occurrences further down) as:
```
# no need to elif here; always returned output above
if type(a) is not mu.ndarray:
    try:
        sum = a.sum
    except AttributeError:
        pass
    else:
        return sum(axis=axis, dtype=dtype, out=out, **kwargs)

return _methods._sum(a, axis=axis, dtype=dtype, out=out, **kwargs)
```